docs/docs/reference-react.md

```diff
@@ -0,0 +1,149 @@
+---
+id: reference-react
+title: React Top-Level API
+permalink: docs/reference-react.html
+next: reference-react-dom.html
+redirect_from: "/docs/reference.html"
+---
+
+`React` is the entry point to the React library. If you're using one of the prebuilt packages it's available as a global; if you're using CommonJS modules you can `require()` it.
+
+## Reference
+
+ - [`React.Component`](#react.component)
+ - [`createClass()`](#createclass)
+ - [`createElement()`](#createlement)
+ - [`cloneElement()`](#cloneelement)
+ - [`createFactory()`](#createfactory)
+ - [`isValidElement()`](#isvalidelement)
+ - [`PropTypes`](#proptypes)
+ - [`Children`](#children)
+
+### `React.Component`
+
+```javascript
+class Greeting extends React.Component {
+  render() {
+    return <h1>Hello, {this.props.name}</h1>;
+  }
+}
+```
+
+`React.Component` is the base class for React components when they are defined using ES6 classes. See the [React.Component API Reference](/react/docs/reference-react-component.html) for a list of methods are provided by the base `React.Component` class.
```

Comment:
This makes it sound like lifecycle hooks (which is what most of that doc is dedicated to) are provided on it whereas the opposite is truth. Maybe say "related to the base ..."?